src/firefly/tweak.py

```diff
@@ -0,0 +1,21 @@
+from firefly import display
+
+
+class TweakDisplay(display.FireflyDisplay):
```

Comment:
Is it possible to record the initial value and then click it goes back to it after tweaking many steps?

Reply:
Yes, but might take more effort. I've opened an issue: https://github.com/spc-group/haven/issues/237